### DISCUSSION TITLE
Meeting: June 18, 2024 agenda

### DISCUSSION THREAD

**Discussion by earth2marsh** [OAI/sig-moonwalk#134] [Announcements]
## Weekly meetings happen on Tuesdays

**Time:** 5pm GMT / 9am Pacific
**Zoom:** [https://zoom.us/j/975841675](https://zoom.us/j/975841675?pwd=SUh4MjRLaEFKNlI3RElpWTdhRDVVUT09), dial-in passcode: 763054

### Accessibility & Etiquette

* Participants must abide by our [Code-of-Conduct](https://github.com/OAI/OpenAPI-Specification/blob/main/CODE_OF_CONDUCT.md#code-of-conduct).
* Meetings are recorded for future reference, and for those who are not able to attend in-person.
* We invited challenges to language or behaviour that is harmful or not inclusive during this meeting.
* We look forward to your participation, but please consider these acts of etiquette:
  * Remain on mute when not speaking to prevent interruptions.
  * Blur your background to reduce visual distractions.
  * Use the Zoom meeting "Raise Hand" feature to notify the group when you wish to speak.

| Blur My Background | Raise Hand |
|-|-|
| <img width="323" alt="Screenshot of Zoom UI showing the 'Stop Video' and 'Blur My Background' control" src="https://github.com/OAI/OpenAPI-Specification/assets/7367/7e43dbbb-6529-46e6-8b04-4c1aa852d9dd"> | <img width="323" alt="Screenshot of Zoom UI showing the 'Reaction' and 'Raise Hand' control" src="https://github.com/OAI/OpenAPI-Specification/assets/7367/f991722f-4651-40aa-9bc4-7e9a2a165a6a"> |

### Agenda Structure

| Topic | Owner | Decision/NextStep |
|-------|---------|---------|  
| | | |
Intros and governance meta-topics (5 mins) | SIG members | -- |
General discussion topics (list below) | SIG members | -- |
Wrap-up and seed the next agenda (5 mins) | SIG members | -- |

Whether attending or not, **anyone can comment on this issue prior to the meeting to suggest topics or to add comments on planned topics or proposals**.

**Comment by earth2marsh** [comment]
Topics proposed from the [June 4th meeting](https://github.com/OAI/sig-moonwalk/discussions/133):

- [Tags](https://github.com/OAI/sig-moonwalk/discussions/67) @lornajane 
- Review the [Respec document](https://oai.github.io/sig-moonwalk/moonwalk.html#introduction)

**Reply by handrews** [reply]
I've been thinking about a high-level functional area organization of the spec, and at the moment here's what I have:

* parsing, manipulating, and serializing OpenAPI Descriptions
* resolving import/reference IRIs to URLs and retrieving them
* modeling data
* modeling http interactions (API shapes)
* securing an API
* deploying an API
* documenting an API for human consumption

Note that establishing a base IRI and resolving relative IRI-references to IRIs is in the first bullet point; the second is only about mapping identifiers to locations and retrieving resource representations from locations.  These two bullet points should be implemented as separate components (with many tools outsourcing the second bullet, as they outsource to reference "resolvers"/removers today).  The first bullet point also includes a bundled single-document form.

**Reply by lornajane** [reply]
After today's discussion, I think this is a good structure. How these pieces interact with one another is still open to discussion. My personal reflection is that security and deployments should remain independent of one another.

**Comment by earth2marsh** [comment]
Video here: https://zoom.us/rec/share/WhbmAV436O_2JV97W0qWt9Ifa8OC4CJKrjUdKBFWHteVjiJXCzyABO9Qv09poPqe.vFTr8bMW54ztb0Ez
